### PR TITLE
docs(): add sources summary page

### PR DESCRIPTION
**Pull request by laulpogan**
## Checklist

- [X] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))

This is an integrations page for the DH docs. See it live here: https://laulpogan.github.io/datahubSitePreview/integrations/

Nice to haves:
Pull metadata from docusaurus json files / tag pages etc with frontmatter like tags
Review tags for quality.

### CODE REVIEW

File: docs-website/src/pages/docs/sources-summary.jsx [hunk 1]
```diff
@@ -0,0 +1,728 @@
+import React from "react";
+import Layout from "@theme/Layout";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import FilterBar from "./_components/FilterBar";
+import FilterCards from "./_components/FilterCards";
+import useGlobalData from "@docusaurus/useGlobalData";
+import DropDownFilter from "./_components/DropDownFilter";
+
+const metadata = [
```
Comment (hsheth2) [inline docs-website/src/pages/docs/sources-summary.jsx:9]: can we put this in JSON and then `import` it?

**Review by hsheth2 — COMMENTED** [review]
It'd be great to update this page https://datahubproject.io/docs/cli/#sources

Could you also put a screenshot of the page in the PR description to give us an idea of what it looks like?

I don't really understand all the jsx/css, but it seems like a lot of stuff. I also saw some algolia things that didn't seem fully related to this change?

File: docs-website/src/pages/docs/sources-summary.jsx (hunk 1 above)
Comment (laulpogan) [reply docs-website/src/pages/docs/sources-summary.jsx]: yeah, will put into a son file

**Comment by laulpogan** [comment]
> It'd be great to update this page https://datahubproject.io/docs/cli/#sources
Will take a look

> Could you also put a screenshot of the page in the PR description to give us an idea of what it looks like?
https://laulpogan.github.io/datahubSitePreview/integrations/ do ya one better! This is slightly laggin since it needs to be pushed to, but it's a live version to play with

> I don't really understand all the jsx/css, but it seems like a lot of stuff. I also saw some algolia things that didn't seem fully related to this change?
The component is initally copied from the search bar component, so we can chop out the irrelevant styles- it didn't feel particularly relevant, but happy to do it

**Comment by laulpogan** [comment]
@hsheth2 all requested changes made!

File: docs-website/package.json [hunk 2]
```diff
@@ -21,15 +21,18 @@
...
     "@docusaurus/preset-classic": "^2.1.0",
     "@octokit/plugin-retry": "^3.0.9",
     "@octokit/plugin-throttling": "^3.5.1",
     "@octokit/rest": "^18.6.2",
+    "antd": "^5.0.7",
     "clsx": "^1.1.1",
     "docusaurus-graphql-plugin": "0.5.0",
     "docusaurus-plugin-sass": "^0.2.1",
     "dotenv": "^16.0.1",
+    "fs": "^0.0.1-security",
```
Comment (laulpogan) [inline docs-website/package.json:35]: @hsheth2 should I be worried about this? I don't think this looks correct 😬

Comment (hsheth2) [reply docs-website/package.json:35]: yes that doesn't look right - we should be able to remove that since `fs` is built-in

**Review by hsheth2 — APPROVED** [review]
LGTM

**Pull request merged**